frontend/src/stories/Card.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import { ImageCard } from "../components/card/ImageCard";
+import { MultiStory } from "./common/MultiStory";
+
+export default {
+  title: "Example/Image Card",
+  component: ImageCard,
+  parameters: {
+    layout: "centered",
+  },
+} as ComponentMeta<typeof ImageCard>;
+
+const Template: ComponentStory<typeof ImageCard> = (args) => (
+  <MultiStory
+    stories={[
+      {
+        label: "Normal",
+        story: <ImageCard {...args} />,
+      },
+      {
+        label: "with unsupported file formal",
+        story: (
+          <ImageCard
+            {...{
+              ...args,
+              asset: {
+                ...args.asset,
+                attributes: {
+                  ...args.asset.attributes,
+                  origin_path: "/amsterdam.mp4",
+                },
+              },
+            }}
+          />
```

Comment:
Just FYI you can also write this as
```suggestion
          <ImageCard
            {...args}
            asset={{
              ...args.asset,
              attributes: {
                ...args.asset.attributes,
                origin_path: "/amsterdam.mp4",
              },
            }}
          />
```